### PR TITLE
chore: update docs with new functionality

### PR DESCRIPTION
**Pull request by wswebcreation**
This PR will update the README with the new added functionality for webviews and the new selectors for iOS

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -209,6 +209,38 @@ representing the ability to take certain actions in the Inspector:
...
+
+![Inspector window for HTML](docs/images/screen-inspector-browser-dom.png).
+
+As of Appium Desktop version [`1.8.0`](https://github.com/appium/appium-desktop/releases/) it will automatically enable showing the HTML-source if a Chrome or Safari session is started.
+
+#### More selector options
+XML is not a native programming language for iOS as it is with Android. Appium will automatically translate the iOS-UIHierarchy into XML, but this is a time-consuming process. 
+The XML will be used to provide a XPATH that can be used to find elements. When you use that XPATH selector with iOS during automation, Appium always needs to do the translation which will slow down the test execution (for more info see [Appium Pro newsletter 8](https://appiumpro.com/editions/8-how-to-find-elements-in-ios-not-by-xpath)).
+ 
+As of Appium Desktop version [`1.8.0`](https://github.com/appium/appium-desktop/releases/) it will, if possible, also provide the:
```
Comment (KazuCocoa) [inline README.md:229]: `1.18.0`?

File: README.md [hunk 2]
```diff
@@ -209,6 +209,38 @@ representing the ability to take certain actions in the Inspector:
...
+It will automatically determine for Android or iOS if a webview is detected and will notify you about this in the right part of the screen.
+
+![Webview selection](docs/images/webview-selection.png)
+
+A select box of all found contexts will be shown and after selecting a context the source will be updated and show the source.
+The below screenshot shows the HTML source of the Appium Desktop documentation website
+
+![Inspector window for HTML](docs/images/screen-inspector-browser-dom.png).
+
+As of Appium Desktop version [`1.8.0`](https://github.com/appium/appium-desktop/releases/) it will automatically enable showing the HTML-source if a Chrome or Safari session is started.
```
Comment (KazuCocoa) [inline README.md:223]: `1.18.0`?

File: README.md (hunk 1 above)
Comment (wswebcreation) [reply README.md]: Yeah, didn't you know? 😛 

Just changed it, thanks!

File: README.md (hunk 2 above)
Comment (wswebcreation) [reply README.md]: Fixed it

**Pull request merged**